src/config.h.in

```diff
@@ -53,6 +53,8 @@
 #define alloca _alloca
 #elif __MINGW32__
```

Comment:
Should these be `#ifdef/#if defined()` instead of just `#if` or does CMake generate 0 or 1 for all of them automatically?